_sass/mixins/_type-mixins.scss

```diff
@@ -0,0 +1,86 @@
+// heading() mixin
+//
+// Use the heading() mixin to add bundles of 
+// typographical styles to a selector
+//
+// Styleguide mixins.heading
+@mixin heading($level) {
+  margin: 0 0 1rem 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 0 1rem` instead of `0 0 1rem 0`